3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,69 @@
+const yearOfBirth = 1987;
+const yearFuture = 2045;
+
+const age = yearFuture - yearOfBirth;
+
+console.log("You will be " + age + " years old in " + yearFuture + ".");
+//2
+
+const dogYearOfBirth = 2010;
+const dogYearFuture = 2045;
+const shouldShowResultInDogYears = true;
+
+const dogAgeInHumanYears = dogYearFuture - dogYearOfBirth;
+const dogAgeInDogYears = shouldShowResultInDogYears
+  ? dogAgeInHumanYears * 7
+  : dogAgeInHumanYears;
+
+if (shouldShowResultInDogYears) {
+  console.log(
+    "Your dog will be " +
+      dogAgeInDogYears +
+      " dog years old in " +
+      dogYearFuture +
+      "."
+  );
+} else {
+  console.log(
+    "Your dog will be " +
+      dogAgeInHumanYears +
+      " human years old in " +
+      dogYearFuture +
+      "."
+  );
+}
+//3
+
+const peterWidth = 8;
+const peterDepth = 10;
+const peterHeight = 10;
+const peterGardenSize = 100;
+const peterActualPrice = 2500000;
+
+const peterVolumeInMeters = peterWidth * peterDepth * peterHeight;
+
+const peterExpectedPrice =
+  peterVolumeInMeters * 2.5 * 1000 + peterGardenSize * 300;
+
+if (peterActualPrice > peterExpectedPrice) {
```

Comment:
since you started to use ternary operator in first exercise, maybe you can try it here as well? code will look more structured :)
E.g. `console.log(peterActualPrice > peterExpectedPrice ? "Peter is paying too much for the house." : "Peter is getting a good deal for the house.");
`